test/js/components/ReadableDate-spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { assert } from 'chai';
+import { shallow } from 'enzyme';
+
+import { ReadableDate } from '../../../src/js/components';
```

Comment:
At `${pastJob}` we just put the tests in the same directory as the regular code, and then skipped over it when disting / bundling. Personally I much prefer that method, so if you're happy to change the globs in the gulpfile to skip everything that ends in -spec, I'm happy to go that way. I don't much care for the `../../../../../../../../../../../foo/bar/baz/Widget.js` imports either :D